proto/sge/bet/bet.proto

```diff
@@ -104,6 +104,9 @@ message Bet {
 
     // the bet is settled
     STATUS_SETTLED = 6;
+
+    // the bet placement failed
+    STATUS_FAILED = 7;
```

Comment:
where is this being used?

Reply:
This is available in oracle, I have put it to be the same